stylelint.config.js

```diff
@@ -6,6 +6,14 @@ module.exports = {
   // https://stylelint.io/user-guide/configuration
   rules: {
     'at-rule-no-unknown': null,
-    'scss/at-rule-no-unknown': true
+    'scss/at-rule-no-unknown': true,
+    // To ignore valid ::v-deep selector:
+    // https://github.com/bencodezen/vue-enterprise-boilerplate/issues/190
```

Comment:
I believe we can remove this comments. The code should already be explicit enough.

```suggestion
```